src/grpc/utils.hpp

```diff
@@ -29,4 +29,36 @@ static bool get_file_contents(const std::string& file_name, std::string& content
     return false;
 }
 
-} // namespace sisl::grpc
+[[maybe_unused]] static void serialize_to_byte_buffer(io_blob_list_t const& cli_buf, grpc::ByteBuffer& cli_byte_buf) {
+    folly::small_vector< grpc::Slice, 4 > slices;
+    for (auto const& blob : cli_buf) {
+        slices.emplace_back(blob.cbytes(), blob.size(), grpc::Slice::STATIC_SLICE);
+    }
+    cli_byte_buf.Clear();
+    grpc::ByteBuffer tmp(slices.data(), cli_buf.size());
+    cli_byte_buf.Swap(&tmp);
+}
+
+[[maybe_unused]] static grpc::Status try_deserialize_from_byte_buffer(grpc::ByteBuffer const& cli_byte_buf,
```

Comment:
I think we should remove maybe_unused since there is no error message or assert in this fuction if status is not okay. it can help caller to detect issue if error is not caught?

Reply:
same as above, caller actually checks the return value